react/features/speaker-stats/components/AbstractSpeakerStatsList.js

```diff
@@ -80,7 +84,10 @@ const abstractSpeakerStatsList = (speakerStatsItem: Function, itemStyles?: Objec
     }, [ faceExpressions ]);
 
     const localSpeakerStats = Object.keys(speakerStats).length === 0 ? getSpeakerStats() : speakerStats;
-    const userIds = Object.keys(localSpeakerStats).filter(id => localSpeakerStats[id] && !localSpeakerStats[id].hidden);
+    const localSortedSpeakerStatsIds
+        = sortedSpeakerStatsIds.length === 0 ? Object.keys(localSpeakerStats) : sortedSpeakerStatsIds;
+
+    const userIds = localSortedSpeakerStatsIds.filter(id => localSpeakerStats[id] && !localSpeakerStats[id].hidden);
```

Comment:
This is the entry point of our new changes. We are using the new `state['features/speaker-stats'].sortedSpeakerStatsIds` state to determine ordering and will continue to use `state['features/speaker-stats'].stats` to hold speaker stats data